### PR TITLE
Upgrade requirements and installed apps for updated xblocks

### PR DESCRIPTION
**Pull request by saharm1**
@stvstnfrd

### CODE REVIEW

File: requirements/edx/stanford.txt [hunk 1]
```diff
@@ -16,3 +16,4 @@ ubcpi-xblock==0.5.3
 -e git+https://github.com/oppia/xblock.git@9f6b95b7eb7dbabb96b77198a3202604f96adf65#egg=oppia-xblock
 git+https://github.com/mfogel/django-settings-context-processor.git
 git+https://github.com/open-craft/problem-builder.git@v2.6.5#egg=xblock-problem-builder==2.6.5
+-e /edx/app/edxapp/xblocks/xblock-submit-and-compare
```
Comment (stvstnfrd) [inline requirements/edx/stanford.txt]: @saharm1 This line should actually replace line 8 above. While we're installing our local copy, we do _not_ want to install the current/master version.

Comment (caesar2164) [reply requirements/edx/stanford.txt]: Why is this change in this PR in the first place, @stvstnfrd ?

Comment (saharm1) [reply requirements/edx/stanford.txt]: @caesar2164 this was only so I could show @stvstnfrd my progress and check what wasnt working. Ill close it now.

**Comment by saharm1** [comment]
@stvstnfrd upgraded versions!

**Pull request merged**